Shaders/PostProcess/ScreenSpaceReflection/private/ComputeIntersection.fx

```diff
@@ -238,14 +238,37 @@ float ValidateHit(float3 Hit, float2 ScreenCoordUV, float3 RayDirectionWS, float
     return Vignette * Confidence;
 }
 
+// Samples a normal from Visible Normal Distribution as described in
+// [1] "Sampling Visible GGX Normals with Spherical Caps" (2023) by [Jonathan Dupuy, Anis Benyoub]
+//     https://arxiv.org/pdf/2306.05044.pdf
+
+float3 SmithGGXSampleVisibleNormalExt(float3 View, // View direction in tangent space
```

Comment:
Let's move this function to `PBR_Common.fxh` along with `SmithGGXSampleVisibleNormal` and name it `SmithGGXSampleVisibleNormalSC`